mops/src/opsa/opsa.md

```diff
@@ -0,0 +1,49 @@
+
+# Outer Product Scatter-Add
+
+## Inputs
+
+$K$ is a dense matrix of floats, expected to be very large in one dimension ($N$), but finite/fixed in the the other ($F$).
```

Comment:
we might not want to use $N$ both here and in the output since the sizes would be different
```suggestion
$K$ is a dense matrix of floats, expected to be very large in one dimension ($M$), but finite/fixed in the the other ($F$).
```

Reply:
Should we use the same names as the code and call the inputs `A` and `B` and the output `C`?

Reply:
For the multi-body order accumulation, we will have different arrays where naming as A_1, A_2 and so on will be more appropriate. If we had only 2 matrices to accumulate, then I'd agree A & B would be alright.

Reply:
Put different sizes for inputs and output.